### PR TITLE
refactor: Reformatted files for better readability.

### PR DESCRIPTION
**Pull request by Sai-Suraj-27**
# PR Description 
Reformatted files for better readability and according to latest Python standards by using **f-strings** instead of the old literal formatting method. To do this I used the automated tool [Flynt](https://github.com/ikamensh/flynt)

## Checklist 
- [ ] Did you add a function?
- [ ] Did you add the tests?
- [ ] Did you follow the steps we provided?

### Socials: 
https://twitter.com/Sai_Suraj_27

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/nn/functional/extension.py [hunk 1]
```diff
@@ -1 +0,0 @@
-
```
Comment (vedpatwardhan) [inline ivy/functional/frontends/paddle/nn/functional/extension.py:1]: could you please revert this change?

File: ivy/functional/frontends/paddle/nn/functional/input.py [hunk 2]
```diff
@@ -1 +0,0 @@
-
```
Comment (vedpatwardhan) [inline ivy/functional/frontends/paddle/nn/functional/input.py:1]: and this too?

File: ivy_tests/conftest.py [hunk 3]
```diff
@@ -55,7 +55,7 @@ def pytest_terminal_summary(terminalreporter):
         return
 
     passed_ratio = 1 - (session.testsfailed / session.testscollected)
-    text = " {:.1%} of {} passed ".format(passed_ratio, session.testscollected)
+    text = f" {passed_ratio:.1%} of {session.testscollected} passed "
```
Comment (vedpatwardhan) [inline ivy_tests/conftest.py:58]: I think the `%` sign should be removed from here

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_input.py [hunk 4]
```diff
@@ -1 +0,0 @@
-
```
Comment (vedpatwardhan) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_input.py:1]: could you please revert this change?

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_signal.py [hunk 5]
```diff
@@ -1 +0,0 @@
-
```
Comment (vedpatwardhan) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_signal.py:1]: and this too?

**Review by vedpatwardhan — CHANGES_REQUESTED** [review]
Hey @Sai-Suraj-27, just requested a few changes, I'll also request @KareemMAX's review. Thanks 😄

**Comment by Sai-Suraj-27** [comment]
@vedpatwardhan sir, sorry for the delay. I made the required changes, can you please review now, Thank you.

File: ivy_tests/conftest.py (hunk 3 above)
Comment (Sai-Suraj-27) [reply ivy_tests/conftest.py:56]: @vedpatwardhan sir, I think it should be present. Please correct me if I am wrong.

Comment (vedpatwardhan) [reply ivy_tests/conftest.py:56]: sorry my bad, yeah we should retain the `%` 👍

**Comment by vedpatwardhan** [comment]
Hey @Sai-Suraj-27, seems like quite a few tests are failing in the CI, could you please pull from `main` in your branch? Would provide a better idea of whether there are test failures related to your changes. Also, I think @RickSanchezStoic could take it from here, I mistakenly thought the review was intended whereas it was auto-requested due to modifications of some files. Thanks @Sai-Suraj-27 @RickSanchezStoic 😄

**Comment by Sai-Suraj-27** [comment]
@vedpatwardhan, @RickSanchezStoic sir, both the main tests `display_test_results` and `The lint / Check formatting / check-formatting` tests are successful. Can you please check now, Thank you.

File: automation_tools/dashboard_automation/update_db.py [hunk 6]
```diff
@@ -22,9 +22,10 @@
 
 
 def make_clickable(url, name):
-    return '<a href="{}" rel="noopener noreferrer" '.format(
-        url
-    ) + 'target="_blank"><img src={}></a>'.format(name)
+    return (
+        '<a href="{}" rel="noopener noreferrer" '.format(url)
```
Comment (KareemMAX) [inline automation_tools/dashboard_automation/update_db.py:26]: Why this is still here?

File: ivy/data_classes/container/base.py [hunk 7]
```diff
@@ -3883,7 +3881,7 @@ def _align_arrays(str_in):
                             rep = tuple(
                                 [
                                     (
-                                        "{} = {}".format(name, v[i])
+                                        f"{name} = {v[i]}"
                                         if v[i].size < self._print_limit
                                         else "{} = {}, shape={}".format(
```
Comment (KareemMAX) [inline ivy/data_classes/container/base.py:3886]: Why this is still here?

File: run_tests.py [hunk 8]
```diff
@@ -36,9 +36,10 @@
 
 
 def make_clickable(url, name):
-    return '<a href="{}" rel="noopener noreferrer" '.format(
-        url
-    ) + 'target="_blank"><img src={}></a>'.format(name)
+    return (
+        '<a href="{}" rel="noopener noreferrer" '.format(url)
```
Comment (KareemMAX) [inline run_tests.py:40]: Why this is still here?

File: run_tests_CLI/array_api_run_tests.py [hunk 9]
```diff
@@ -11,9 +11,10 @@
 
 
 def make_clickable(url, name):
-    return '<a href="{}" rel="noopener noreferrer" '.format(
-        url
-    ) + 'target="_blank"><img src={}></a>'.format(name)
+    return (
+        '<a href="{}" rel="noopener noreferrer" '.format(url)
```
Comment (KareemMAX) [inline run_tests_CLI/array_api_run_tests.py:15]: Why this is still here?

File: scripts/eager_mode_benchmark/benchmark.py [hunk 10]
```diff
@@ -344,4 +344,4 @@ def visualize_speed_up(
                     "backend={} and device={}".format(backend, device)
```
Comment (KareemMAX) [inline scripts/eager_mode_benchmark/benchmark.py:344]: Why this is still here?

**Review by KareemMAX — CHANGES_REQUESTED** [review]
Hey @Sai-Suraj-27,
This change looks very useful to me, thank you for having the time to look into it. I've pointed out some places were the automation seems that it didn't work. Maybe my list is not really exhaustive.

Also, I feel like it's worth adding a pre-commit hook for this. To make sure these changes are enforced all the time (and also fixed by our lint-bot)

**Comment by Sai-Suraj-27** [comment]
> Hey @Sai-Suraj-27, This change looks very useful to me, thank you for having the time to look into it. I've pointed out some places were the automation seems that it didn't work. Maybe my list is not really exhaustive.
> 
> Also, I feel like it's worth adding a pre-commit hook for this. To make sure these changes are enforced all the time (and also fixed by our lint-bot)

@KareemMAX sir, I made all the requested changes, please check now. Regarding the addition of a pre-commit hook, I'm not sure if this [Flynt](https://github.com/ikamensh/flynt) tool works perfectly, because it is not able to detect and change the formatting to f-strings at every place. Also, i see they mentioned a [few potential errors](https://github.com/ikamensh/flynt#dangers-of-conversion) that can happen while using this. So, i guess instead of adding it as a pre-commit hook, its better to run it at some regular intervals, review the changes done, and then merge. Please tell me your opinion on this, thank you.  
image

**Comment by KareemMAX** [comment]
@Sai-Suraj-27 I see, I would of course love having us using `f""` over `format()` mainly because of code readability concerns. And ideally the [potential errors](https://github.com/ikamensh/flynt#dangers-of-conversion) you mentioned shouldn't happen as those are bad practices that might cause those issues (where `'%s'.format()` behaves different than `__repr__`).

What do you think about that automation @vedpatwardhan? Do you think the benefits outweigh the risks? Since you are more familiar with how are codebase look like in detail I would trust your say on that.

**Comment by vedpatwardhan** [comment]
> @Sai-Suraj-27 I see, I would of course love having us using `f""` over `format()` mainly because of code readability concerns. And ideally the [potential errors](https://github.com/ikamensh/flynt#dangers-of-conversion) you mentioned shouldn't happen as those are bad practices that might cause those issues (where `'%s'.format()` behaves different than `__repr__`).
> 
> What do you think about that automation @vedpatwardhan? Do you think the benefits outweigh the risks? Since you are more familiar with how are codebase look like in detail I would trust your say on that.

Yeah it would be a useful automation as long as it doesn't get too complicated. Thanks @Sai-Suraj-27 😄
